drivers/tty/serdev/pibridge.c

```diff
@@ -0,0 +1,453 @@
+#include <linux/errno.h>
+#include <linux/kfifo.h>
+#include <linux/module.h>
+#include <linux/mutex.h>
+#include <linux/of.h>
+#include <linux/serdev.h>
+#include <linux/pibridge_comm.h>
+#include <linux/wait.h>
+
+#define PIBRIDGE_BAUDRATE		115200
+#define PIBRIDGE_IO_TIMEOUT		10         // msec
+#define PIBRIDGE_RECV_BUFFER_SIZE	100
+#define PIBRIDGE_BC_ADDR		0xff
+
+#define PIBRIDGE_RESP_CMD		0x3fff
+#define PIBRIDGE_RESP_OK		0x4000
+#define PIBRIDGE_RESP_ERR		0x8000
+
+struct pibridge {
+	struct serdev_device *serdev;
+
```

Comment:
Is this blank line intentional?

Reply:
I guess sometimes a little grouping in structs doesn't hurt. ;)